src/viam/app/data_client.py

```diff
@@ -174,33 +174,48 @@ async def tabular_data_by_filter(
         return data
 
     async def binary_data_by_filter(
-        self,
-        filter: Optional[Filter] = None,
-        dest: Optional[str] = None,
+        self, filter: Optional[Filter] = None, dest: Optional[str] = None, include_file_data: bool = True, num_files: Optional[int] = None
     ) -> List[BinaryData]:
         """Filter and download binary data.
 
         Args:
-            filter (viam.proto.app.data.Filter): Optional `Filter` specifying binary data to retrieve. No `Filter` implies all binary data.
-            dest (str): Optional filepath for writing retrieved data.
+            filter (Optional[viam.proto.app.data.Filter]): Optional `Filter` specifying binary data to retrieve. No `Filter` implies al
```

Comment:
```suggestion
            filter (Optional[viam.proto.app.data.Filter]): Optional `Filter` specifying binary data to retrieve. No `Filter` implies all
```

Reply:
@stuqdog do you want to make the change yourself and merge now that Bashar is gone?